tgui/packages/tgui/styles/components/Button.scss

```diff
@@ -76,6 +76,13 @@ $bg-map: colors.$bg-map !default;
   }
 }
 
+.Button--hasContent.Button--iconPosition--right {
```

Comment:
You forgot to add this class to button, smth like:
```
iconPosition && 'Button--iconPosition--' + iconPosition
```

Reply:
Fixed. Button icons now align properly, main post updated.